src/vmod_saintmode.vcc

```diff
@@ -91,7 +104,10 @@ Used for assigning the backend from the saintmode object.
 
 Example::
 
-    sub vcl_backend_fetch {
-        set bereq.backend = sm.backend();
```

Comment:
I think the complete example serves better to show that part, rather than the method example.